.browserslistrc

```diff
@@ -0,0 +1 @@
+defaults
```

Comment:
add this file to .gitignore and run command: `git rm --cached .browserlistrc` to untrack it

Reply:
May I know how to deal with it?

Reply:
okie
Step 1: Open file: `.gitignore` and add this line: `.browserlistrc` (same as line 44 in bellow picture)
Step 2: Open terminal and cd to your project. Then run command: `git rm --cached .browserlistrc`
That's all !
![image](https://user-images.githubusercontent.com/65896888/89747650-ee643080-dae9-11ea-8d55-3c7f143687da.png)

Reply:
Done it. Thank you.